bin/reth/src/args/rpc_server_args.rs

```diff
@@ -78,6 +89,76 @@ impl RpcServerArgs {
             }
         }
     }
+
+    pub(crate) async fn server<Client, Pool, Network>(
```

Comment:
this needs docs which include the info that it spawns the server and returns the handle. See launch docs.

and should be named `start_server` or `launch_server` instead.